app/include/zmk/leader.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+void zmk_leader_activate(int32_t timeout, bool timeout_on_activation, uint32_t position);
+void zmk_leader_deactivate();
+bool zmk_leader_get_status();
```

Comment:
`status` I don't love... is active? is enabled? Something a bit more aligned with the bool return value here.